### DISCUSSION TITLE
Streaming partitioned parquet

### DISCUSSION THREAD

**Discussion by GREEB** [perspective-dev/perspective#2105] [Q&A]
Hello again,

We are looking at a way to reduce memory usage when streaming arrow files (starlette).

Seems like Parquet partitioned files are not supported, is it possible to use partitioned parquet files? if it is, could it be implemented without losing sorting/filtering functionality.
 
Thanks for any help

**Comment by texodus** [comment]
I do not understand how parquet support is related to memory usage.  If you're having memory issues you suspect are out of line, please provide a repro so we can analyze it.
